docs/release/releases/release-1.8.md

```diff
@@ -0,0 +1,38 @@
+# Cluster API v1.8
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.8` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 22nd April 2024      | week 1   |
+| *v1.6.x & v1.7.x released* <sup>[1]</sup>            | Release Lead | Tuesday 23rd April 2024     | week 1   |
```

Comment:
For Kubernetes we only need v1.7.x. So I would suggest to drop v1.6 here (as we don't cherry-pick 1.30 support into 1.6)